src/legacy/core_plugins/metrics/public/components/aggs/moving_average.js

```diff
@@ -135,29 +156,30 @@ const MovingAverageAggUi = props => {
       <EuiFlexGroup gutterSize="s">
         <EuiFlexItem>
           <EuiFormRow
-            id={htmlId('model')}
-            label={(<FormattedMessage
-              id="tsvb.movingAverage.modelLabel"
-              defaultMessage="Model"
-            />)}
+            id={htmlId('model_type')}
+            label={i18n.translate('tsvb.movingAverage.modelLabel', {
+              defaultMessage: 'Model',
+            })}
           >
             <EuiComboBox
               isClearable={false}
-              placeholder={intl.formatMessage({ id: 'tsvb.movingAverage.model.selectPlaceholder', defaultMessage: 'Select' })}
+              placeholder={i18n.translate('tsvb.movingAverage.model.selectPlaceholder', { defaultMessage: 'Select' })}
               options={modelOptions}
               selectedOptions={selectedModelOption ? [selectedModelOption] : []}
-              onChange={handleSelectChange('model')}
+              onChange={handleSelectChange('model_type')}
               singleSelection={{ asPlainText: true }}
             />
           </EuiFormRow>
         </EuiFlexItem>
         <EuiFlexItem>
           <EuiFormRow
             id={htmlId('windowSize')}
-            label={(<FormattedMessage
-              id="tsvb.movingAverage.windowSizeLabel"
-              defaultMessage="Window Size"
-            />)}
+            label={i18n.translate('tsvb.movingAverage.windowSizeLabel', {
+              defaultMessage: 'Window Size',
+            })}
+            helpText={!isProperWindowValue(model) && i18n.translate('tsvb.movingAverage.windowSizeHint', {
+              defaultMessage: 'Window must be more than twice the size of the period',
```

Comment:
> your window must always be at least twice the size of your period

Reply:
Another thing to nice is that this checks needs to be specific for Holt Winters, not for every function

Reply:
Nice catch! Done